suites/pacific/cephfs/tier-0_fs_kernel.yaml

```diff
@@ -133,10 +133,16 @@ tests:
       abort-on-fail: false
   - test:
       name: Run fsstress on kernel and fuse mounts
-      module: cephfs_bugs.test_fsstress_on_kernel_and_fuse.py
+      module: cephfs_bugs.fsstress_kernel_verification.py
```

Comment:
any reason for having a separate test just for kernel verification just for pacific ?